server/services/gdrive/gdrive.hooks.ts

```diff
@@ -0,0 +1,37 @@
+import Ajv from 'ajv';
+import { gDriveSchema } from '../../../client/schemas/gdrive';
+import {AjvOrNewable, validateSchema} from "feathers-hooks-common";
+
+const ajv = new Ajv({ allErrors: true, $data: true });
+
+export default {
+  before: {
+    all: [],
+    find: [],
+    get: [],
+    create: [validateSchema(gDriveSchema, <AjvOrNewable>ajv)], // TODO: Validate Permissions
```

Comment:
If we're gonna have the schema validation, can you put it on the update and patch actions as well?